libs/ui/popover/src/lib/trigger/popover-trigger.directive.ts

```diff
@@ -132,6 +136,12 @@ export class TsPopoverTriggerDirective implements OnInit, OnDestroy, OnChanges,
   }
   public _position = TsPopoverPositions.Bottom;
 
+  /**
+   * Set trigger type of popover, i.e. click or hover
+   */
+  @Input()
+  public showTrigger: TsTrigger = 'click';
```

Comment:
Hmm this name confuses me a bit as it sounds like the 'trigger' is what will be shown. Maybe just `trigger`? Or `popoverTrigger`? or `triggerEvent` etc?

Reply:
Yeah it's meant to mean how trigger shown. Thought about `trigger` but felt it was generic. `popoverTrigger` might be a better choice.